### PR TITLE
Nathandegraaf master

### PR DESCRIPTION
**Pull request by NathanDeGraaf**
Adding a couple new verification checks to our cli in order to test the process of getting my code into master.

### CODE REVIEW

File: src/constants/constants.js [hunk 1]
```diff
@@ -82,7 +82,11 @@ export default {
 
     license: {
       name: 'license',
-      message: 'License:'
+      message: 'License:',
+
+      error: {
+        invalidMessage: 'Invalid Lincense!'
```
Comment (jackcmeyer) [inline src/constants/constants.js]: Spelling mistake.

**Comment by NathanDeGraaf** [comment]
Fixed

**Comment by jackcmeyer** [comment]
Two things: 

1. Can we move those validate functions to their own functions and then write tests for them?
2. Don't forget to add your name to author in package.json

**Comment by jackcmeyer** [comment]
Should we validate github username and password, too?

**Pull request merged**